app/controllers/password_reset_controller.rb

```diff
@@ -0,0 +1,28 @@
+class PasswordResetController < ApplicationController
+
+  def create
+    user = User.find_by(email: params[:email])
+    user.send_password_reset if user
```

Comment:
@rishabhguptarishi if user wasn't found then also you will redirect user with message that password reset instructions are sent on email